fluentd/Dockerfile.centos7

```diff
@@ -4,7 +4,7 @@ MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 ENV DATA_VERSION=1.6.0 \
     FLUENTD_VERSION=0.12.43 \
-    GEM_HOME=/opt/app-root/src \
+    GEM_HOME=/usr/share/gems \
```

Comment:
Both in the 3.11 and master branch, it is set to /opt/app-root/src.  Are they also need to be fixed?
Dockerfile.centos7:    GEM_HOME=/opt/app-root/src \

Reply:
I don't know - 3.11 downstream is working as is - not sure why